src/ensembl/src/content/app/browser/chromosome-navigator/ChromosomeNavigator.scss

```diff
@@ -0,0 +1,45 @@
+@import 'src/styles/common';
+
+.chromosomeNavigator {
+  svg {
+    width: 100%;
+    overflow: visible;
+  }
+
+  .stick {
+    height: 4px;
+    fill: $light-grey;
+    stroke: $grey;
+  }
+
+  .viewport {
+    fill: white;
+  }
+
+  .viewportBorder {
+    fill: none;
+    stroke: $dark-grey;
+  }
+
+  .focusPointer {
+    fill: #ff9800; // FIXME: variable?
+
+    line {
+      stroke: #ff9800;
+    }
+  }
+
+  .centromereRegion {
+    fill: $light-grey;
+    stroke: $light-grey;
+  }
+
+  .centromereCentre {
+    fill: $dark-grey;
+  }
+
+  .label {
+    font-family: 'IBM Plex Mono';
```

Comment:
Can we add variables for the font family too? So next time the font family changes, we don't need to change them everywhere.

Reply:
✔️